distributed-job-manager/src/main/kotlin/ru/fix/distributed/job/manager/annotation/JobIdField.kt

```diff
@@ -0,0 +1,11 @@
+package ru.fix.distributed.job.manager.annotation
+
+/**
+ * For String field in DistributedJobs, which represents id of that job
+ * If field with that annotation won't be found, for job id will be used full class name
+ *
+ * Now you can change that behaviour by overriding {@link ru.fix.distributed.job.manager.DistributedJob}
+ * */
+@Target(AnnotationTarget.FIELD)
+@Retention(AnnotationRetention.RUNTIME)
+annotation class JobIdField
```

Comment:
I'm not sure about this name. There is `JobId` class already in DJM. The `Id` annotaion is used by too many libraries.